localstack/services/awslambda/invocation/version_manager.py

```diff
@@ -275,11 +395,20 @@ def invocation_loop(self) -> None:
                                 self.function_arn,
                             )
                             return
+
+                        # TODO: untested assumption(!)
+                        #   because provisioned concurrency affects the unreserved
```

Comment:
What exactly is the untested assumption here?

If I set my provisioned_concurrency to 50, and reserved_concurrency to 70, I could invoke 120 parallel lambdas as it is currently implemented, right?

Reply:
On the same function it should actually still only be max 70 that you can invoke concurrently. The provisioned concurrency counts towards the reserved concurrent executions of the function

Reply:
So the reason we're not tracking them here is because provisioned concurrency is always automatically deducted in full from the available concurrency (either Account/Region global or per function if it has reserved concurrency set).